components/new-contribution-flow/ContributionFlowPublicMessage.js

```diff
@@ -121,14 +118,7 @@ const ContributionFlowPublicMessage = ({ order, publicMessage }) => {
                 </Flex>
                 <Flex width={1 / 2} alignItems="center" justifyContent="flex-end">
                   <StyledButton buttonSize="tiny" loading={isSubmitting} type="submit" onSubmit={handleSubmit}>
-                    {submitted ? (
-                      <Flex>
-                        <FormattedMessage id="contribute.publicMessage.posted" defaultMessage="Posted" />
-                        <Check size={12} />
-                      </Flex>
-                    ) : (
-                      <FormattedMessage id="contribute.publicMessage.post" defaultMessage="Post message" />
-                    )}
+                    <FormattedMessage id="contribute.publicMessage.post" defaultMessage="Post message" />
```

Comment:
Do we provide any feedback that the message has been sent successfully?

Reply:
Confettis 😉 🎉